pkg/reconciler/workload/resource/resource_controller.go

```diff
@@ -243,53 +254,57 @@ func processNext(
 
 // key is gvr::KEY
 func (c *Controller) processResource(ctx context.Context, key string) error {
+	logger := klog.FromContext(ctx)
 	parts := strings.SplitN(key, "::", 2)
 	if len(parts) != 2 {
-		klog.Errorf("Error parsing key %q; dropping", key)
+		logger.Info("error parsing key; dropping")
 		return nil
 	}
 	gvrstr := parts[0]
+	logger = logger.WithValues("groupVersionResource", gvrstr)
```

Comment:
these long names are really ugly, especially for things that have well established acronyms like `gvr`, `gr`, `gvk`...

Reply:
`s/groupVersionResource/gvr/g`
`s/groupVersionKind/gvk/g`

updated